api/src/main/java/org/openmrs/api/context/Context.java

```diff
@@ -567,6 +567,7 @@ private static Session getMailSession() {
 					props.setProperty("mail.from", adminService.getGlobalProperty("mail.from"));
 					props.setProperty("mail.debug", adminService.getGlobalProperty("mail.debug"));
 					props.setProperty("mail.smtp.auth", adminService.getGlobalProperty("mail.smtp_auth"));
+					props.setProperty(OpenmrsConstants.GP_MAIL_SMTP_STARTTLS_ENABLE, adminService.getGlobalProperty(OpenmrsConstants.GP_MAIL_SMTP_STARTTLS_ENABLE));
```

Comment:
I said that you need to create a separate TRUNK ticket to address because it's a bug in core, then create a separate PR to openmrs:master instead to get it in, this change should not be in the branch for your GSoC project

Reply:
Ok i did it already this was just to permit me do my testing and ensure it works with respect to my gsoc project. pull request for the issue is https://github.com/openmrs/openmrs-core/pull/2703

Reply:
Then remove it please

Reply:
Ok